npc/events/easter_2010.txt

```diff
@@ -309,6 +285,32 @@ prontera,202,297,3	script	Rina#Easter	4_F_JOB_HUNTER,{
 	npcskill "AL_INCAGI",1,0,0;
 	npcskill "AL_BLESSING",1,0,0;
 	end;
+
+L_HuntingInfo:
+	mes "[Rina]";
+	mes "What you need to do is";
+	mes "to choose one monster among";
+	mes "^FF0000DEVIRUCHI,^000000 ^FF0000WRAITH DEAD,^000000";
+	mes "^FF0000DULLAHAN,^000000 ^FF0000NIGHTMARE TERROR^000000";
+	mes "and then kill ^0000FF50^000000 monsters";
+	mes "and come back to me.";
+	next;
+	mes "[Rina]";
+	mes "It must be a tough task,";
+	mes "but you are the right person";
+	mes "who did the oath-taking ceremony.";
+	mes "I hope you succeed.";
+	if (.@quest1 < 2) {
```

Comment:
i think the value of `.@quest1` are always `0` here.
You probably have to pass in the value as `callsub` argument to use it here.